reactapp/src/index.css

```diff
@@ -326,6 +326,21 @@ input[type='checkbox'] {
   margin-right: auto;
 }
 
-.blue-side-notes {
-  
-}
\ No newline at end of file
+.button {
```

Comment:
You may need another button class.  You can keep this one for the solid color buttons, but for the buttons that only become solid color after hovering them, I think you'd need another class.  You can do ".outline-button" or something like that.  Then you can add the hover effect to fill the background color with something like ".outline-button:hover".